Tools/PowershellModule/src/Library/Add-WinGetManifest.ps1

```diff
@@ -21,29 +21,29 @@ Function Add-WinGetManifest
     Name of the Azure Function that hosts the rest source.
 
     .PARAMETER Path
-    The Path to the JSON manifest file or folder hosting the JSON / YAML files that will be uploaded to the rest source. 
-    This path may contain a single JSON / YAML file, or a folder containing multiple JSON / YAML files. Does not support 
-    targetting a single folder of multiple different applications in *.yaml format.
+    The Path to the Application Manifest file or folder hosting either a JSON or YAML file(s) that will be uploaded to the rest source. 
+    This path may contain a single Application Manifest file, or a folder containing files for a single Application Manifest. Does not support 
+    targetting a single folder of multiple different applications.
```

Comment:
Lets be specific about the types of files supported.  This is just ambigious.

Reply:
"either a JSON or YAML file(s)" is ambigious?

Reply:
The second sentence talks about a folder for multiple files.  This is only relevant for yml formatted data.  There is an implicit assumption the reader is familiar with then different data formats and options and what each format implies.

Reply:
```suggestion
    targeting a single folder of multiple different applications.
```